### PR TITLE
fixes #7808 Add learn more link for sticky targeting on Audience page

### LINKED ISSUE #7808: Add learn more link for sticky targeting on Audience page
Followup to #7488 

image

We should add a “learn more” link to an FAQ for enrollment or targeting.

Link: https://experimenter.info/workflow/custom-audiences

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2831)

### PR DESCRIPTION
**Pull request by cherish2003**
### Because
Add learn more link for sticky targeting on Audience page 
### This commit
- Added `FormLabel`  embedded the `Formcheck` inside it so that it will align beside each other
- Added custom styles to added margin by the class `custom-audiences-external-link` in the index.scss file
- Added the external link in the `constants.ts`
### Screenshots
Screenshot from 2023-03-25 01-11-25-1

### CODE REVIEW

**Comment by cherish2003** [comment]
@yashika-khurana Hi mam kindly review the code and let me know any changes are required

File: experimenter/experimenter/nimbus-ui/src/styles/index.scss [hunk 1]
```diff
@@ -163,4 +163,7 @@ select.form-control.is-warning {
     border-color: #28a745;
     padding-right: calc(0.8em + 2.8rem);
     background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='24' width='24' viewBox='0 0 20 20' aria-hidden='true' focusable='false' class='css-tj5bde-Svg'%3E%3Cpath fill='lightgray' d='M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z'%3E%3C/path%3E%3C/svg%3E") right 0.60rem center/20px 25px no-repeat,#fff url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='6' height='6' viewBox='0 0 8 8'%3e%3cpath fill='%2328a745' d='M2.3 6.73L.6 4.53c-.4-1.04.46-1.4 1.1-.8l1.1 1.4 3.4-3.8c.6-.63 1.6-.27 1.2.7l-4 4.6c-.43.5-.8.4-1.1.1z'/%3e%3c/svg%3e") center right 2rem/calc(0.8em + 0.3rem) calc(0.8em + 0.3rem) no-repeat !important;
+}
+.custom-audiences-external-link{
+  margin-left:0.3rem;
 }
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/styles/index.scss:169]: you can use bootstrap css classes to manage the margins- https://getbootstrap.com/docs/4.0/utilities/spacing/

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
Thanks @cherish2003 in general looks good, small suggestion to avoid custom CSS and use bootstrap class to manage margin

File: experimenter/experimenter/nimbus-ui/src/styles/index.scss (hunk 1 above)
Comment (cherish2003) [reply experimenter/experimenter/nimbus-ui/src/styles/index.scss:169]: @yashikakhurana okay mam i will modify the changes get back to you soon..

File: experimenter/experimenter/nimbus-ui/src/styles/index.scss [hunk 2]
```diff
@@ -163,4 +163,4 @@ select.form-control.is-warning {
     border-color: #28a745;
     padding-right: calc(0.8em + 2.8rem);
     background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='24' width='24' viewBox='0 0 20 20' aria-hidden='true' focusable='false' class='css-tj5bde-Svg'%3E%3Cpath fill='lightgray' d='M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z'%3E%3C/path%3E%3C/svg%3E") right 0.60rem center/20px 25px no-repeat,#fff url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='6' height='6' viewBox='0 0 8 8'%3e%3cpath fill='%2328a745' d='M2.3 6.73L.6 4.53c-.4-1.04.46-1.4 1.1-.8l1.1 1.4 3.4-3.8c.6-.63 1.6-.27 1.2.7l-4 4.6c-.43.5-.8.4-1.1.1z'/%3e%3c/svg%3e") center right 2rem/calc(0.8em + 0.3rem) calc(0.8em + 0.3rem) no-repeat !important;
-}
\ No newline at end of file
+}
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/styles/index.scss]: @cherish2003 can you please remove this line, it seems extra space

**Review by yashikakhurana — CHANGES_REQUESTED** [review]

File: experimenter/experimenter/nimbus-ui/src/styles/index.scss (hunk 2 above)
Comment (cherish2003) [reply experimenter/experimenter/nimbus-ui/src/styles/index.scss]: @yashikakhurana mam i have removed the extra space kindly check it

File: experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ import {
   TOOLTIP_DURATION,
 } from "src/lib/constants";
 import { getStatus } from "src/lib/experiment";
+import "src/styles/index.scss";
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx:22]: you will also need to remove this import

**Review by yashikakhurana — CHANGES_REQUESTED** [review]

File: experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx (hunk 3 above)
Comment (cherish2003) [reply experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx]: Mam done with removing the imports

**Review by yashikakhurana — APPROVED** [review]
thank you @cherish2003 🎉

**Pull request merged**